lib/grpc/GrpcServer.ts

```diff
@@ -70,6 +85,56 @@ class GrpcServer {
       });
     });
   }
+
+  private generateCertificate = (tlsCertPath: string, tlsKeyPath: string) => {
+    const keys = pki.rsa.generateKeyPair(1024);
+    const cert = pki.createCertificate();
+
+    cert.publicKey = keys.publicKey;
+    cert.serialNumber = String(Math.floor(Math.random() * 1024) + 1);
+
+    const date = new Date();
+    cert.validity.notBefore = date;
+    cert.validity.notAfter = new Date(date.getFullYear() + 1, date.getMonth(), date.getDay());
```

Comment:
We're making it valid for a year, will we need a validity check to regenerate once it expires? How did you choose a year?

Reply:
I wasn't sure about that one so I checked what LND is doing and they are making the certificate valid for 14 months. So I just made it valid for a year.

But maybe we should remove remove the validity completely for now? @sangaman

Reply:
Yeah, if you can make it valid indefinitely (or like 5+ years) for now that's probably fine. You can also add a TODO here to revisit how we want to handle expired certificates perhaps.